app/src/app/admin/articles/article/article.ts

```diff
@@ -70,7 +70,7 @@ namespace app.admin.articles.article {
                         return articleService.getArticle($stateParams.permalink)
                             .then((article) => {
 
-                                article._articleMetas = articleService.hydrateMetaFromTemplate(article._articleMetas, ArticleConfig.articleMetaTemplate);
+                                article._articleMetas = articleService.hydrateMetaFromTemplate(article, ArticleConfig.articleMetaTemplate);
```

Comment:
Ah ok I see the rationale, but yea I think it would be better to have the article id passed as a param.